G/GTPSA/build_tarballs.jl

```diff
@@ -0,0 +1,65 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GTPSA"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/mattsignorelli/gtpsa.git", "ed2c14eb096a9ecac3f483f9914c0c679765496d")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd gtpsa/
+if [[ $target == *"-apple-"* ]]
+then
+CC=gcc
+CXX=g++
+ln /workspace/destdir/lib/liblapack32.dylib /workspace/destdir/lib/liblapack.dylib
+elif [[ $target == *"linux"* ]]
+then
+ln /workspace/destdir/lib/liblapack32.so /workspace/destdir/lib/liblapack.so
+fi
+cmake . -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN%.*}_gcc.cmake -DCMAKE_BUILD_TYPE=Release
+make
+make install
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("x86_64", "macos"; ),
+    Platform("aarch64", "macos"; ),
+    Platform("i686", "windows"; ),
+    Platform("x86_64", "windows"; )
+]
```

Comment:
It looks like you are basically building for all platforms here, right? If so, just condense it down into this statement.
```suggestion
platforms = supported_platforms()
```

Reply:
FreeBSD is not included in that list - the source code is unable to compile on this platform with the error: ‘__builtin_tgmath’ is not a function pointer . I have had difficulty resolving this and decided, given the purpose/audience/use cases of this package, and compatibility of packages in the same realm as this one, that FreeBSD support is not necessary.

Reply:
In that case, you can use the following code:
```julia
# The code does not compile on FreeBSD due to an error with __builtin_tgmath
platforms = supported_platforms()
filter!(!Sys.isfreebsd, platforms)
```